doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -38,7 +38,9 @@ IDE, OS, and tool support
 Board support
 =============
 
-|no_changes_yet_note|
+* Added:
+
+  * Support for nRF7002-EB II Wi-Fi shield
```

Comment:
EB-II support is already added in v3.0.0, should we extend to say the new host board support?